app/zapier/people/actions.js

```diff
@@ -52,12 +52,12 @@ const inviteUser = {
      */
     async perform(z, bundle) {
       /**
-     * @param {string} email
-     * @param {Account[]} accounts
-     * @returns {Account|undefined}
-     */
+       * @param {string} email
+       * @param {Account[]} accounts
+       * @returns {Account|undefined}
+       */
       function findUser(email, accounts) {
-        for (let i = 0; i < accounts.length; i++) {
+        for (let i = 0; i < accounts.length; i = i + 1) {

```

Comment:
I agree that `i = i + 1` is better than `i++`, but perhaps `i += 1` is a good compromise between the two.